ctgan/synthesizers/ctgan.py

```diff
@@ -232,6 +233,11 @@ def fit(self, train_data, discrete_columns=tuple()):
                 contain the integer indices of the columns. Otherwise, if it is
                 a ``pandas.DataFrame``, this list should contain the column names.
         """
+        if epochs:
```

Comment:
I would check explicitly for `None` here, so any other value that evaluates to `False` does not enter this branch and later on fails.